benchmarks/frontend/scripts/sweep_k8s/dgd.py

```
@@ -0,0 +1,303 @@
+            .get("services", {})
+            .get("Frontend", {})
+            .get("extraPodSpec", {})
+            .get("mainContainer", {})
+            .get("env", [])
+        )
+        idx = None
+        for i, env_var in enumerate(env_list):
+            if env_var.get("name") == "DYN_TOKENIZER_BACKEND":
+                idx = i
+                break
+    except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)